doc/projectionintro.html

```diff
@@ -1,30 +1,30 @@
 <!-- meta page description: Projections and spatial transformations in GRASS GIS-->
 <h3>Projection management in general</h3>
 
-A GRASS location is referenced with a single projection and coordinate system
-(or unreferenced as XY location). When creating a new location from an existing
+A GRASS project is referenced with a single projection and coordinate system
+(or unreferenced as XY project). When creating a new project from an existing
 raster or vector map using the tools available from the startup screen or
 the map import commands, projection and coordinate system are defined.
-To change the projection of maps, a new location has to be created and
-the desired maps have to be reprojected into it from the source location
+To change the projection of maps, a new project has to be created and
+the desired maps have to be reprojected into it from the source project
 as explained below.
 
 <h3>Raster map projections</h3>
 
 Raster maps are backward projected in GRASS. This means that the
```

Comment:
I find "backward projected" a confusing concept/phrase. I'm not sure I have a better wording in mind because I'm not sure I fully understand the paragraph. Is this about "Raster Map Reprojection"?

Reply:
```suggestion
Rasters are reprojected using the raster projection command (<a href=r.proj.html>r.proj</a>). The command is used in the target project to "pull" a map from its source project. Both projects need to have a projection defined.
```

My best attempt at clarifying this paragraph.

Reply:
I think all that means is pulling the data to you as opposed to pushing them somewhere. The following paragraph uses _forward transformation_. I'm not sure how useful it is to use it, so I dropped "backward projected" from the paragraph, but left "forward transformation".